lib/vm/src/memory.rs

```diff
@@ -27,14 +27,13 @@ pub fn read_region(ctx: &Ctx, ptr: u32) -> Vec<u8> {
     let region = to_region(ctx, ptr);
     let memory = ctx.memory(0);
 
-    // TODO: there must be a faster way to copy memory
     match WasmPtr::<u8, Array>::new(region.offset).deref(memory, 0, region.len) {
         Some(cells) => {
+            // In case you want to do some premature optimization, this shows how to cast a `&'mut [Cell<u8>]` to `&mut [u8]`:
```

Comment:
Good link, but I was surprised to see `iov` all over than code snippet :joy: 
That is really a pretty ugly cast sequence, but if they say it works, it must